diesel/src/mysql/connection/bind.rs

```diff
@@ -846,8 +846,9 @@ mod tests {
             ).execute(conn)
             .unwrap();
 
-        let stmt = conn.prepared_query(&crate::sql_query(
-            "SELECT
+        let stmt = super::super::prepared_query(
```

Comment:
I would prefer `crate::mysql::connection` or something like that here instead of `super::super`.